packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/ZoomOutButton/ZoomOutButtonContainer.js

```diff
@@ -19,16 +19,29 @@ function storeMapper (stores) {
 class ZoomOutButtonContainer extends React.Component {
   constructor () {
     super()
-    this.onClick = this.onClick.bind(this)
+    this.onPointerDown = this.onPointerDown.bind(this)
+    this.onPointerUp = this.onPointerUp.bind(this)
+    this.timer = ''
   }
 
-  onClick () {
-    this.props.zoomOut()
+  onPointerDown (event) {
+    const { zoomOut } = this.props
+    this.timer = setInterval(zoomOut, 100)
+  }
+
+  onPointerUp (event) {
+    clearInterval(this.timer)
```

Comment:
My comments on the `onPointerUp` of the `ZoomInButtonContainer` also applies here.

It's actually possible to create a never-ending Zoom In timer AND a never-ending Zoom Out timer. The resulting conflict causes the Subject image to jitter back and forth between different sizes at epileptic speeds, which makes for a fun, wholesome evening of viewing entertainment for the family. (My Netflix is down, you see.)